lib/models/movie_location.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies_locations',
```

Comment:
Actually, I don't think this model is necessary. Bookshelf models don't have a 1:1 relationship between models and tables. In this case, you will still only need two models, `movie` and `location`. The way you take advantage of the many to many relationship is with [`belongsTo*`](http://bookshelfjs.org/#many-to-many) functionality with bookshelf. This would allow you to use the following code in your API:

```
return new Movie({ id: 'id' }).fetch({ withRelated: 'locations' })
.then((movie) => {
  const locations = movie.related('locations');
  locations.forEach((location) => {
    console.log(location.get('name'));
  });
})
```